pages/result/result.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://crimsonui-beta.netlify.app/css/components.css"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;500;600;700;800&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="../../css/main.css" />
+    <link rel="stylesheet" href="../question/question.css" />
+    <link rel="stylesheet" href="./result.css" />
+    <title>SuperFan Trivia</title>
+  </head>
+  <body>
+    <main class="main-body">
+      <header>
+        <h1 class="main-heading">RESULT</h1>
+      </header>
+      <div class="question-container">
+        <h3 class="final-score">Final Score: <span>20/30</span></h3>
+
+        <!-- Question -->
+        <h3 id="question">What is Floki's profession?</h3>
+        <div class="option-container">
+          <div class="option correct-option">Ship Builder</div>
+          <div class="option wrong-option">Carpenter</div>
+          <div class="option">Blacksmith</div>
+        </div>
+        <!-- Question -->
+        <h3 id="question">
```

Comment:
Use different ids or use a common class.

Reply:
Thanks for pointing it out, I will change it to class.